packages/nextjs/app/builders/0x4e97cE84631F7EF26B43F66a0623F00261EdfD6e/page.tsx

```diff
@@ -0,0 +1,61 @@
+"use client";

```

Comment:
Since we don't have any interactivity on the page, we could remove "use client"  which in turn will mark this as server component and [give some nice benefits](https://nextjs.org/docs/app/building-your-application/rendering/server-components#benefits-of-server-rendering)

Reply:
Made the change